lib/reverse_words.rb

```diff
@@ -1,6 +1,26 @@
-# A method to reverse each word in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time Complexity: O(n) because there is a constant ratio (3n/2) to the size of the input array.
+# Space Complexity: O(1) because no additional memory is needed regardless of input array.
+
 def reverse_words(my_words)
-  raise NotImplementedError
+  i = 0
```

Comment:
Nit: these variable names (`i` and `j`) could be more descriptive/self-explanatory. The rest of your variable names are good though! :smile: